chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -309,6 +309,23 @@ spec:
           - name: secret_mount_path
             value: "/var/secrets"
           {{- end }}
+          {{- if .Values.eventSubscription.auditing.enabled }}
+          {{- if .Values.nats.external.enabled }}
+          - name: nats_address
+            value: "{{ .Values.nats.external.host }}"
+          - name: nats_port
+            value: "{{ .Values.nats.external.port }}"
+          {{- else }}
+          - name: nats_address
+            value: "nats.{{ .Release.Namespace }}.svc.{{ .Values.kubernetesDNSDomain }}"
+          - name: nats_port
+            value: "4222"
+          {{- end }}
+          - name: auditing
```

Comment:
How about only emit this whole section if and only if `.Values.eventSubscription.auditing.enabled`?

I did the same for metering to minimise the YAML on deployments for support purposes.

Reply:
This is already in place, the whole configuration section for NATS and auditing is only included if `.Values.eventSubscription.auditing.enabled`